### DISCUSSION TITLE
Is there a Style guide I can reference?

### DISCUSSION THREAD

**Discussion by jadedrakerider** [docsifyjs/docsify#1841] [Q&A] [answered]
Hello, I am a hobbyist, so I need a little pointing in the right direction. I am trying to write documentation for a project, but I don't seem to be able to find a style guide to tell me how your markup works. I checked out "marked" and their documentation doesn't seem to be about how you actually format a page.  

Can somebody point me in the right direction?

UPDATE:
Turns out marked doesn't actually have its own syntax, it's just a parser of Markdown.  Markdown has [documentation](https://www.markdownguide.org/) one can reference.

**Comment by sy-records** [comment, marked as answer]
Yes, you can write according to Markdown's grammar.
